src/chainconsumer/chain.py

```diff
@@ -444,3 +458,30 @@ class MaxPosterior(BetterBase):
     def vec_coordinate(self) -> np.ndarray:
         """The coordinate as a numpy array, in the order the columns were given."""
         return np.array(list(self.coordinate.values()))
+
+
+def _filter_var_names(var_names: list[str], all_vars: list[str]):
+    """
+    Helper function to return the var_names to allows filtering parameters names.
+    """
+
+    if not var_names:
+        return all_vars
+
+    elif var_names:
```

Comment:
Because the primary `if` has a return, better to simply unindent and remove the `elif` - will keep the code simpler and easier to read